README.md

```diff
@@ -7,23 +7,35 @@ Grouplogger is a specialized Stackdriver logging client for writing groups of lo
 ([Stackdriver documentation: Viewing related request log entries](https://cloud.google.com/appengine/docs/flexible/go/writing-application-logs#related-app-logs))
 
 ```go
-var r *http.Request
-var ctx *context.Context
+// Stackdriver requires a non-nil http.Request.
+u, err := url.Parse("http://notareal.website/search?q=hihihi")
+if err != nil {
+	log.Fatal(err)
+}
+req := &http.Request{
+	Header: http.Header{
+		"X-Cloud-Trace-Context": []string{"your-trace-id"},
+	},
+	URL: u,
+}
 
-cli, err := NewClient(ctx, "parent-id")
+cli, err := grouplogger.NewClient(context.Background(), "your-project-id")
 if err != nil {
-	// TODO: handle.
+	log.Fatal(err)
```

Comment:
I noticed it's common in GoDocs to use the `TODO` pattern for error handling:

```go
if err != nil {
  // TODO: handle.
}
```

I prefer it here because it avoids introducing a second logging package (`log`) in the documentation for a logging package.

Reply:
Ditto lines 13, 38.

Reply:
Good call, this is better. But I just got rid of any pretense at error handling instead to make things shorter, as you suggested in another comment.